src/common/transformations/tests/common_optimizations/weights_dequantize_to_fake_quantize.cpp

```diff
@@ -100,8 +122,21 @@ TEST_P(TranslateNewWeightFormatToOldOne, ReshapeMatMul) {
 INSTANTIATE_TEST_SUITE_P(
     NGraph,
     TranslateNewWeightFormatToOldOne,
-    testing::Combine(testing::Values(FQ_as_Mul_Sub_dequantize{-128, 127, 1, 2, (-128 - 1) * 2, (127 - 1) * 2, 256},
-                                     FQ_as_Mul_Sub_dequantize{-127, 127, 1, 2, (-127 - 1) * 2, (127 - 1) * 2, 255},
-                                     FQ_as_Mul_Sub_dequantize{-128, 127, 0, 2, (-128 - 0) * 2, (127 - 0) * 2, 256},
-                                     FQ_as_Mul_Sub_dequantize{-127, 127, 0, 2, (-127 - 0) * 2, (127 - 0) * 2, 255}),
-                     testing::Values(element::f32, element::f16)));
+    testing::Combine(
+        testing::Values(
+            FQ_as_Mul_Sub_dequantize{-128, 127, ZPType::FLOAT, {.float_val = 1}, 2, (-128 - 1) * 2, (127 - 1) * 2, 256},
+            FQ_as_Mul_Sub_dequantize{-127, 127, ZPType::FLOAT, {.float_val = 1}, 2, (-127 - 1) * 2, (127 - 1) * 2, 255},
+            FQ_as_Mul_Sub_dequantize{-128, 127, ZPType::FLOAT, {.float_val = 0}, 2, (-128 - 0) * 2, (127 - 0) * 2, 256},
+            FQ_as_Mul_Sub_dequantize{-127, 127, ZPType::FLOAT, {.float_val = 0}, 2, (-127 - 0) * 2, (127 - 0) * 2, 255},
+            FQ_as_Mul_Sub_dequantize{-128, 127, ZPType::INT8_T, {.int8_val = 1}, 2, (-128 - 1) * 2, (127 - 1) * 2, 256},
+            FQ_as_Mul_Sub_dequantize{-127, 127, ZPType::INT8_T, {.int8_val = 1}, 2, (-127 - 1) * 2, (127 - 1) * 2, 255},
+            FQ_as_Mul_Sub_dequantize{-128, 127, ZPType::INT8_T, {.int8_val = 0}, 2, (-128 - 0) * 2, (127 - 0) * 2, 256},
+            FQ_as_Mul_Sub_dequantize{-127,
+                                     127,
+                                     ZPType::INT8_T,
+                                     {.int8_val = 0},
+                                     2,
+                                     (-127 - 0) * 2,
+                                     (127 - 0) * 2,
+                                     255}),
```

Comment:
Minor: can we use unified style?
```suggestion
            FQ_as_Mul_Sub_dequantize{-127, 127, ZPType::INT8_T, {.int8_val = 0}, 2, (-127 - 0) * 2, (127 - 0) * 2, 255}),
```

Reply:
If I do this modification clang_format check fails.

Reply:
Maybe I can disable clang-format on this particular place:
![image](https://github.com/openvinotoolkit/openvino/assets/7381433/bee4bdc2-77d1-4d75-b954-69b3deb7608a)

Reply:
> Maybe I can disable clang-format on this particular place: ![image](https://user-images.githubusercontent.com/7381433/269534978-bee4bdc2-77d1-4d75-b954-69b3deb7608a.png)

I agree to do that because in this case the instances are much more readable